stl/inc/xutility

```diff
@@ -987,7 +988,8 @@ namespace ranges {
         public:
             template <class _Ty1, class _Ty2>
                 requires (_Choice<_Ty1, _Ty2>._Strategy != _St::_None)
-            constexpr void operator()(_Ty1&& _Val1, _Ty2&& _Val2) const noexcept(_Choice<_Ty1, _Ty2>._No_throw) {
+            constexpr _STATIC_CALL_OPERATOR void operator()(_Ty1&& _Val1, _Ty2&& _Val2) _CONST_CALL_OPERATOR

```

Comment:
One more occurrence of `constexpr _STATIC_CALL_OPERATOR` contrary to our conventional order.